radio/src/gui/horus/bitmapbuffer.cpp

```diff
@@ -510,7 +510,7 @@ BitmapBuffer * BitmapBuffer::load(const char * filename)
           result = f_read(&bmpFile, (uint8_t *)&pixel, 4, &read);
           if (result != FR_OK || read != 4) {
             f_close(&bmpFile);
-            free(bmp);
+            delete bmp;

```

Comment:
Ouch yes, thanks, I changed my mind when modifying the function.
I am sure it remains a lot of bugs to find, I wrote a lot of code during the latest weeks